warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java

```diff
@@ -312,26 +312,24 @@ protected void addTokenizedContent(IngestHelperInterface helper, RawRecordContai
         if (helper instanceof AbstractContentIngestHelper) {
             String tokenDesignator = Objects.toString(((AbstractContentIngestHelper) helper).getTokenFieldNameDesignator(), "");
             AbstractContentIngestHelper h = (AbstractContentIngestHelper) helper;
-            boolean contentIndexed;
             for (String field : fields.keySet()) {
-                contentIndexed = false;
-                if (h.isContentIndexField(field)) {
-                    updateForIndexedField(helper, event, fields, countDelta, loadDate, tokenDesignator, field);
-                    termFrequencyFieldsInfo.createOrUpdate(field + tokenDesignator, event.getDataType().outputName(),
+                String fieldTokenDesignator = h.isContentIndexField(field) ? tokenDesignator : "";
+                if (h.isContentIndexField(field) || h.isIndexListField(field)) {
+                    updateForIndexedField(helper, event, fields, countDelta, loadDate, fieldTokenDesignator, field);
+                    termFrequencyFieldsInfo.createOrUpdate(field + fieldTokenDesignator, event.getDataType().outputName(),
                                     MetadataWithMostRecentDate.IGNORED_NORMALIZER_CLASS, event.getDate());
-                    contentIndexed = true;
                 }
                 
-                if (h.isReverseContentIndexField(field)) {
-                    updateForReverseIndexedField(helper, event, fields, countDelta, loadDate, tokenDesignator, field);
-                    contentIndexed = true;
+                if (h.isReverseContentIndexField(field) || h.isReverseIndexListField(field)) {
+                    updateForReverseIndexedField(helper, event, fields, countDelta, loadDate, fieldTokenDesignator, field);
                 }
                 
-                if (contentIndexed) {
-                    log.debug("{}{} as a data type", field, tokenDesignator);
```

Comment:
It looks like we are losing the log.debug and the update call here if the field is a 'content index field' and not an 'index field list'.

Reply:
This has changed in the latest commit.